examples/medplum-patient-intake-demo/src/App.tsx

```diff
@@ -27,7 +27,11 @@ export function App(): JSX.Element | null {
         },
         {
           title: 'Upload Data',
-          links: [{ icon: <IconHealthRecognition />, label: 'Upload Example Patient Data', href: '/upload/example' }],
+          links: [
+            { icon: <IconDatabaseImport />, label: 'Upload Core ValueSets', href: '/upload/core' },
+            { icon: <IconQuestionMark />, label: 'Upload Questionnaires', href: '/upload/questionnaire' },
```

Comment:
Please let me know whether it is best to combine those into a single upload or keep them separate.